ksql-functional-tests/src/test/resources/query-validation-tests/elements.json

```diff
@@ -60,6 +60,27 @@
       "inputs": [{"topic": "input", "value": {"c1": 4}}],
       "outputs": [{"topic": "OUTPUT", "value": {"C1": 4}}]
     },
+    {
+      "name": "validate without elements OK - JSON_SR SCHEMA",
+      "statements": [
+        "CREATE STREAM INPUT WITH (kafka_topic='input', value_format='JSON_SR');",
+        "CREATE STREAM OUTPUT WITH(PARTITIONS = 4) as SELECT * FROM input;"
+      ],
+      "topics": [
+        {
+          "name": "input",
+          "schema": {"type": "object","properties": {"c1": {"type": "integer"}}},
+          "format": "JSON_SR"
+        },
+        {
+          "name": "OUTPUT",
+          "format": "JSON_SR",
+          "partitions": 4
+        }
+      ],
+      "inputs": [{"topic": "input", "value": {"c1": 4}}],
+      "outputs": [{"topic": "OUTPUT", "value": {"C1": 4}}]
```

Comment:
If we need the mapper to have `USE_BIG_DECIMAL_FOR_FLOATS` for decimals to work, should we maybe include some doubles and decimals in the test data?

Reply:
there's a test for this in `KsqlJsonSerializerTest` that explicitly tests that but I can try to add one here as well